std_msgs/Header header

std_msgs/String state
int64 target_id
geometry_msgs/Point32 center_of_mass
float32 distance

int64[] track_ids
float32[] confidences

std_msgs/String[] classifier_names
float32[] classifier_confidences
